hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestBlockTokens.java

```diff
@@ -377,6 +386,23 @@ private static void setSecureConfig() throws IOException {
         ozoneKeytab.getAbsolutePath());
   }
 
+  @Test
+  public void testRotateKeySCMAdminCommand()
+      throws InterruptedException, TimeoutException, IOException {
+    GenericTestUtils.waitFor(() -> cluster.getScmLeader() != null, 100, 1000);
+    InetSocketAddress address =
+        cluster.getScmLeader().getClientRpcAddress();
+    String hostPort = address.getHostName() + ":" + address.getPort();
+    String[] args = {"scm", "rotate", "--scm", hostPort};
+
+    String oldKey =
+        scmClient.getSecretKeyClient().getCurrentSecretKey().toString();
```

Comment:
May be we can add a small test here, run twice "getCurrentSecretKey" and assert whether both are same before rotating.